policyengine_us/parameters/gov/local/ca/riv/general_relief/qualified_noncitizen_status.yaml

```diff
@@ -0,0 +1,16 @@
+description: Riverside county considers these statuses as qualified noncitizens under the General Relief Program.
+values:
+  1975-01-01:
+    - REFUGEE
+    - ASYLEE
+    - DEPORTATION_WITHHELD
+    - CONDITIONAL_ENTRANT
+    - LEGAL_PERMANENT_RESIDENT
+    - VICTIMS_OF_TRAFFICKING
```

Comment:
I think I missed this in the AABD program but `VICTIMS_OF_TRAFFICKING` is not a valid immigration status. We should leave as a comment for now